lib/messagebird/callflow.rb

```diff
@@ -0,0 +1,58 @@
+require "messagebird/base"
+
+module MessageBird
+  class CallFlow < MessageBird::Base
+    attr_accessor :id, :title, :record, :steps, :default, :createdAt, :updatedAt
```

Comment:
please specify `attr_reader` for those attributes where writer is already defined in the class.